builds/e2e/longhaul.yaml

```diff
@@ -346,4 +346,128 @@ jobs:
           snitch.storage.account: '$(snitch.storage.account)'
           snitch.storage.masterKey: '$(StorageAccountMasterKeyStress)'
           loadGen.message.frequency: '$(loadGen.message.frequency.amd64)'
-         
\ No newline at end of file
+          
+################################################################################
+  - job: setup_iotuap_amd64_moby
+################################################################################
+    displayName: Windows IoT Core AMD64 Moby (Setup Only)
+    condition: eq(variables['run.iotuap.amd64.moby'], 'true')
+    pool:
+      name: $(pool.name)
+      demands:
+        - longhaul_iotuap_x64 -equals true
+    variables:
+      edgelet.artifact.name: 'iotedged-windows'
+    steps:
+      - checkout: none
+      - task: AzureKeyVault@1
+        displayName: 'Azure Key Vault'
```

Comment:
you don't need this key vault step in this setup job.

Reply:
I think we need that, since we need to get the edgebuilds-azurecr-io username and password used in the last step of this stage:
_- template: templates/deploy-iotuap-setup.yaml_

Reply:
I take that back, not being used. I'll remove that step.